backend/homework/6/src/main/java/org/example/services/ServiceVehicle.java

```diff
@@ -0,0 +1,34 @@
+package org.example.services;
+
+import jakarta.annotation.PostConstruct;
+import org.example.entities.Vehicle;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+@Component
+public class ServiceVehicle {
+    private List<Vehicle> vehicles = new ArrayList<>();
+    public Vehicle findMostExpensiveVehicle(){
+        return vehicles.stream()
```

Comment:
it is possible that the object will be null, so it's important to provide a proper message or handling mechanism for that scenario.